Javascript/research.js

```diff
@@ -275,7 +275,7 @@ resdesc = [null,
     "[6x25] Auto Challenge. Enough said. (Lets you automatically run and complete challenges!)",
     "[7x1] A new row, old upgrade. Makes all ticks 1.2% faster each level.",
     "[7x2] Wow! Industries sponsors another +0.9% per level towards your Ascension bank!",
-    "[7x3] Hey, I totally didn't steal this idea. For each level, you gain 12 blessings of Wow! Cube tier for every Tesseract opened.",
+    "[7x3] Hey, I totally didn't steal this idea. You gain 12 blessings of Wow! Cube tier for every Tesseract opened.",
```

Comment:
Not sure when it was nerfed, but...

![image](https://user-images.githubusercontent.com/31546028/94093154-bff6a680-fe46-11ea-926c-3a48517c74b5.png)

No point saying "For each level" if there's only one level available.